### ISSUE TITLE
Failed to `Running an example Job` in kubernetes document

### ISSUE THREAD

**Issue by XiShanYongYe-Chang** [kubernetes/website#34073]
When I run a `Job` following to the example in the [kubernetes document](https://kubernetes.io/docs/concepts/workloads/controllers/job/#running-an-example-job), the pod generated by the job fails to run:

```log
Can't use an undefined value as an ARRAY reference at /usr/local/lib/perl5/5.36.0/Math/BigInt/Calc.pm line 1049.
```

This should be due to the release of the v5.36.0 of the `perl` image,  The previous version v5.34.0 image I tried worked fine.

**Comment by XiShanYongYe-Chang** [comment]
@JohnTitor @sftim Thanks~

**Comment by JohnTitor** [comment]
Note that I've already opened #34070.

**Comment by dorisyang-forever** [comment]
Same problem. 
Use perl image 5.34.0, it works!

**Comment by sftim** [comment]
This needs a fix that is similar to https://github.com/kubernetes/website/pull/34475

**Comment by jihoon-seo** [comment]
@sftim it seems that the PR #34475 itself fixed this issue, am I right?

**Comment by jihoon-seo** [comment]
Resolved by #34414.
/close
